akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -476,19 +477,20 @@ private[remote] class Association(
                 clearOutboundCompression()
                 clearInboundCompression(u)
                 // end delivery of system messages to that incarnation after this point
-                send(ClearSystemMessageDelivery, OptionVal.None, OptionVal.None)
+                send(ClearSystemMessageDelivery(current.incarnation), OptionVal.None, OptionVal.None)
```

Comment:
This confused me a little at first. Just to clarify. We send this message down the outbound stream but SystemMessageDelivery intercepts it, clears its seqNr and then pulls another message?